checker/src/main/java/org/checkerframework/checker/guieffect/GuiEffectTypeFactory.java

```diff
@@ -55,6 +55,8 @@
      */
     protected final Set<LambdaExpressionTree> uiLambdas = new HashSet<>();
 
+    private final Set<TypeElement> uiAnonClasses = new HashSet<>();
```

Comment:
Some javadoc similar to that of `uiLambdas` above would probably be useful.
Should this field also be `protected` for consistency with `uiLambdas`?

Reply:
Done in next commit.